devices/iluminize.js

```diff
@@ -124,6 +124,13 @@ module.exports = [
         description: 'ZigBee 3.0 LED-controller, 4 channel 5A, RGBW LED',
         extend: extend.light_onoff_brightness_colortemp_color(),
     },
+    {
+        zigbeeModel: ['HK-ZD-RGB-A', '5110.40'],
+        model: '5110.40',
+        vendor: 'Iluminize',
+        description: 'Zigbee 3.0 LED Controller Multi 5 - 4A,RGB W/CCT LED',
+        extend: extend.light_onoff_brightness_colortemp_color(),
```

Comment:
Please add the color temperature range as described on https://www.zigbee2mqtt.io/how_tos/how_to_support_new_devices.html#31-retrieving-color-temperature-range-only-required-for-lights-which-support-color-temperature

Reply:
ok, commit is available now.